src/main/java/pl/edu/mimuw/queue/IntQueueNode.java

```diff
@@ -3,8 +3,8 @@
 public class IntQueueNode {
 
   private final int value;
-  private final IntQueueNode next;
-  private final IntQueueNode prev;
+  public IntQueueNode next;
```

Comment:
w tym przypadku to praktyczne bardzo, ale pamiętaj że settery dają Ci dodatkowa swobode - zawsze mozesz dodac logike jakas przy ustawianiu wartosc, bez potrzeby edycji miejsc w ktorych sa uzywane, a tak trzeba by tysiace linijek przerabiac 🙃